### DISCUSSION TITLE
"github.com/account/organizations/new" have very interesting bug.

### DISCUSSION THREAD

**Discussion by NnicanBuak** [community/community#47868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

## After quite a while choosing a name to create an organization I got the error "Too many requests" and my home page opened under the name input field, as I understand I had to be returned to the main page with an error.

> ### And so I was able to create recursion by reopening the organization creation in the created window:

![github com_account_organizations_new_plan=free ref_cta=Create%2520a%2520free%2520organization ref_loc=cards ref_page=%2Forganizations%2Fplan](https://user-images.githubusercontent.com/89499782/219857526-7a4baf11-6724-43f6-8d75-662b9b4e677a.png)

---
<sub>Browser: Google Chrome 110.0.5481.104 
OS: Windows 11</sub>

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @NnicanBuak ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
